src/main/java/org/prebid/cache/handlers/GetCacheHandler.java

```diff
@@ -50,6 +51,7 @@ public GetCacheHandler(final ReactiveRepository<PayloadWrapper, String> reposito
                            final PrebidServerResponseBuilder builder,
                            final CircuitBreaker webClientCircuitBreaker) {
 
+        super(new ConditionalLogger(log));
```

Comment:
Pls make ConditionalLogger part of other local variables, don't pass it to parent